tutorials/preprocessing/plot_80_boxy_processing.py

```diff
@@ -0,0 +1,331 @@
+"""
```

Comment:
>  no further differences in the pipeline would be necessary after this import function

If this is the case then we probably don't need another tutorial with an example pipeline.

However, if there are differences for processing frequency domain NIRS data then perhaps a frequency domain processing example would be appropriate that specifically highlights the differences between continuous and frequency domain analysis. I would try and keep it generic rather than tied to the specific ISS vendor format though (just looking at the filename).

Reply:
yes can rename or remove after, just using this as a testbed to see if my functions work

Reply:
and yes I will reduce the reliance on vendor specific naming once I have things working